internal/modules/slashing/test.py

```diff
@@ -0,0 +1,74 @@
+import time
+import logging
+import json
+import unittest
+from utils import exec_command, env
+from core.keys import keys_show
+from modules.staking.query import query_validator, fetch_validator_pubkey_from_node
+from modules.slashing.query import *
+from modules.slashing.tx import (
+    tx_unjail,
+)
+
+HOME = env.HOME
+NODE2_HOME = env.get("NODE2_HOME")
+NODE3_HOME = env.get("NODE3_HOME")
+
+cmd = f"sudo -S systemctl stop simd-3"
+exec_command(cmd)
+time.sleep(20)
+
+validator = keys_show("validator3", "val", NODE3_HOME)[1]["address"]
```

Comment:
Let's check if `NUM_VALS` are `>=3` for testing slashing. Otherwise this will just fail. Error a note in-case if `NUM_VALS < 3`